.github/workflows/deploy-to-azure-app-service.yml

```diff
@@ -247,6 +254,33 @@ jobs:
             -WebAppName ${{ inputs.app-name }} `
             -SlotName ${{ inputs.slot-name }}
 
+      - name: Fetch tags
+        id: fetch-tags
+        run: |
+          git fetch --tags
+
+      - name: Determine timestamp
+        id: determine-timestamp
+        run: |
+          # Get the current timestamp in the format of yyyy-MM-dd-HH.mmUTC - can't use : in tags
```

Comment:
Is this a standard timestamp format, something that other programs can understand by default? If not, please use one that is.

Reply:
This is not entirely in a correct existing format, that's because of limitations of git tags not allowing certain characters (most formats use ":" for time notations) - So now it's ISO 8601 but with ":" replaced by "." other formats might work better but are much less human readable

Reply:
Can you select a timestamp format that's standard and can also be used in a Git tag? We might want to process these values later.

Reply:
You can use `YYYY-MM-DDTHH-MM-SS`.

Reply:
This is not the timestamp format being used.